dub.selections.json

```diff
@@ -1,7 +1,7 @@
 {
 	"fileVersion": 1,
 	"versions": {
-		"dub": "1.34.0",
+		"dub": {"repository":"git+https://github.com/symmetryinvestments/dub.git","version":"a892208154301797838def4cbd2ba47d1af3da5b"},
```

Comment:
Just for CI tests; this is dub v1.34 + one commit (https://github.com/symmetryinvestments/dub/commit/a892208154301797838def4cbd2ba47d1af3da5b), the one we are currently using at Symmetry.